src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figure {
+    public static final String defaultColor = "WHITE";
+    private String color;
+
+    public void drawArea() {
+
+    }
```

Comment:
If `Figure` is meant to be a base class for other figures, consider making `drawArea()` an abstract method. This will enforce that all subclasses implement their own version of `drawArea()`.